com.github.Flacon.yaml

```diff
@@ -0,0 +1,220 @@
+app-id: com.github.Flacon
+runtime: org.kde.Platform
+runtime-version: "5.13"
+sdk: org.kde.Sdk
+command: flacon
+rename-desktop-file: flacon.desktop
+rename-icon: flacon
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --share=ipc
+  
```

Comment:
You need to give it access to device=dri as I'm getting this when I run it 
```
libEGL warning: wayland-egl: could not open /dev/dri/card0 (No such file or directory)
```

Reply:
Added